<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoiceAssistant - Pre alpha!</string>
  </property>
  <widget class="QWidget" name="container">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Loading â€¦</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="muteButton">
        <property name="text">
         <string>Mute</string>
        </property>
        <property name="icon">
         <iconset theme="audio-input-microphone"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="textLabel">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="content">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>478</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionCloseWindow"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Models">
    <property name="title">
     <string>Voice &amp;recognition</string>
    </property>
    <addaction name="actionHas_word"/>
    <addaction name="actionOpen_downloader"/>
   </widget>
   <widget class="QMenu" name="menuCommands">
    <property name="title">
     <string>&amp;Commands</string>
    </property>
    <addaction name="actionAddCommand"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Models"/>
   <addaction name="menuCommands"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionHas_word">
   <property name="text">
    <string>Can recognize word</string>
   </property>
  </action>
  <action name="actionOpen_downloader">
   <property name="text">
    <string>Open downloader</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAddCommand">
   <property name="text">
    <string>Add command</string>
   </property>
  </action>
  <action name="actionCloseWindow">
   <property name="text">
    <string>Close window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
